apps/web/components/connected/DrivingInstructorList/DrivingInstructorList.tsx

```diff
@@ -0,0 +1,193 @@
+import { useQuery } from '@apollo/client'
+import {
+  AlertMessage,
+  Box,
+  Input,
+  LoadingDots,
+  Pagination,
+  Table as T,
+  Text,
+} from '@island.is/island-ui/core'
+import {
+  ConnectedComponent,
+  GetDrivingInstructorsQuery,
+  GetDrivingInstructorsQueryVariables,
+} from '@island.is/web/graphql/schema'
+import { useNamespace } from '@island.is/web/hooks'
+import { GET_DRIVING_INSTRUCTORS_QUERY } from '@island.is/web/screens/queries/DrivingInstructors'
+import { useState } from 'react'
+
+const DEFAULT_ITEMS_PER_PAGE = 10
+
+type DrivingInstructor = GetDrivingInstructorsQuery['drivingLicenseTeachersV4'][number]
+
+const getSortedAndFilteredDrivingInstructors = (
+  instructors: DrivingInstructor[],
+  searchValue: string,
+): DrivingInstructor[] => {
+  const searchTerms = searchValue
+    .replace('´', '')
+    .trim()
+    .toLowerCase()
+    .split(' ')
+
+  const fullSearchString: string = searchTerms.join(' ')
+  const brokersStartingWithFullSearchString: DrivingInstructor[] = []
+  const brokersContainingAllTerm: DrivingInstructor[] = []
+
+  const startsWithFullSearchString = (
+    instructor: DrivingInstructor,
+  ): boolean => {
+    return (
+      instructor.name?.trim().toLowerCase().startsWith(fullSearchString) ||
+      instructor.nationalId?.trim().toLowerCase().startsWith(fullSearchString)
```

Comment:
Does `toLowerCase` make sense for national IDs?

Reply:
You are right, I'll take it out, thanks for the comment. Classic mindless copy paste :)